app/models/host_status/execution_status.rb

```diff
@@ -1,17 +1,21 @@
 class HostStatus::ExecutionStatus < HostStatus::Status
+  # execution finished successfully
   OK = 0
```

Comment:
We should never use that integer directly, it's just storage value so why we'd store them as symbol/text? All host statuses are stored as numbers in Foreman (and this state already used OK = 0 and ERROR = 1) I just added two new.